### ISSUE TITLE
features.py get_text_features pandas 1x support

### ISSUE THREAD

**Issue by sykesdev** [ScottfreeLLC/AlphaPy#33]
the following lines...

```
    min_length = int(feature.str.len().min())
    max_length = int(feature.str.len().max())
```

seem to fall foul of Pandas 1.x stricter rules. The following seems to work...

```
    min_length = int(feature.astype(str).str.len().min())
    max_length = int(feature.astype(str).str.len().max())
```

**Comment by mrconway** [comment]
Thanks, we'll release an update this weekend.

**Comment by mrconway** [comment]
Fixed in [2.4.3](https://github.com/ScottfreeLLC/AlphaPy/releases/tag/2.4.3)
